src/main/java/seedu/bigpp/command/viewercommand/ViewerDeleteCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.bigpp.command.viewercommand;
+
+import seedu.bigpp.command.Command;
+import seedu.bigpp.pc.PC;
+import seedu.bigpp.pc.PCList;
+
+public class ViewerDeleteCommand extends Command {
+
+    public ViewerDeleteCommand(String arguments) {
+        super.setArguments(arguments);
+    }
+
+    /**
+     * Delete a PC of a given index from the PC list
+     *
+     * @return if the index is a Prebuilt PC, inform the user that the PC of that index cannot be deleted
+     * @return the name of the PC deleted
+     */
+    @Override
+    public String executeCommand() {
+        int index = Integer.parseInt(super.getArguments()) - 1;
+        PC pc = (PCList.getList()).get(index);
+        if (pc.getIsPreBuilt()) {
```

Comment:
Good handling of case where user is trying to delete a prebuilt pc(cannot be changed or deleted)

Reply:
Keep this structure for this method in future implementations